packages/server/validators/guilds.js

```diff
@@ -0,0 +1,219 @@
+    .exists({ checkFalsy: true })
+    .withMessage("Guild handler's name can't be null or empty.")
+    .trim()
+    .isLength({ min: 1, max: 255 })
+    .withMessage("Guild handler's name must be between 1 to 255 characters."),
+
+  // Description Check
+  body("description", "Invalid guild description.")
+    .exists({ checkFalsy: true })
+    .withMessage("Guild description can't be null or empty.")
+    .trim()
+    .isLength({ max: 2000 })
+    .withMessage("Guild description max length is 2,000 characters."),
+
+  // Category Check
+  body("category", "Invalid guild category.")
+    .exists({ checkFalsy: true })
+    .withMessage("Guild category can't be null or empty.")
+    .trim()
+    .isLength({ max: 255 })
+    .withMessage("Category name max length is 255 characters."),
+
+  // Location Check: OPTIONAl FOR GUILD CREATION
+  body("location", "Invalid guild location.")
+    .optional()
+    .trim()
+    .isLength({ max: 255 })
+    .withMessage("Location name max length is 255 characters."),
+
+  // Website Check: OPTIONAL FOR GUILD CREATION
+  body("website", "Invalid guild website.")
+    .optional()
+    .trim()
+    .isLength({ max: 2048 })
+    .withMessage("Guild thumbnail max length is 2048 characters.")
+    .matches(/^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/)
+    .withMessage("Provided website url is invalid."),
+
+  // Tags Check
+  body("tags", "Invalid tags.").isArray().withMessage("Tags must be an array."),
+
+  // Banner Check
+  body("banner", "Invalid guild banner.")
+    .exists({ checkFalsy: true })
+    .withMessage("Guild banner can't be null or empty.")
+    .trim()
+    .isLength({ max: 255 })
+    .withMessage("Guild thumbnail max length is 255 characters.")
+    .matches(/.*\.(jpg|png|JPG|PNG|JPEG|jpeg)$/)
+    .withMessage("Uploaded file is not a valid image."),
+
+  // Avatar Check
+  body("avatar", "Invalid guild avatar.")
+    .exists({ checkFalsy: true })
+    .withMessage("Guild avatar can't be null or empty.")
+    .trim()
+    .isLength({ max: 255 })
+    .withMessage("Guild thumbnail max length is 255 characters.")
+    .matches(/.*\.(jpg|png|JPG|PNG|JPEG|jpeg)$/)
+    .withMessage("Uploaded file is not a valid image."),
+
+  // Media Check
+  body("media", "Invalid guild media.")
+    .isArray()
+    .withMessage("Media must be an array."),
+
+  // IsInviteOnly Check
+  body("isInviteOnly", "Invalid guild invitation mode.")
+    .isBoolean()
+    .withMessage("Invite mode must be set to true or false."),
+];
+
+const updateGuildValidator = [
+  param("guildId", "Invalid Guild ID.")
+    .exists({ checkFalsy: true })
+    .withMessage("Guild ID can't be null or empty.")
+    .trim()
+    .escape()
+    .isUUID()
+    .withMessage("Guild ID not a valid UUID.")
+    .custom(async (value) => {
+      try {
+        await GuildController.getGuild(value);
+      } catch (error) {
+        throw new Error(`No guild exists with an ID of ${value}`);
+      }
+    }),
```

Comment:
You can get rid of this duplicate `guildId` parameter field check since you already have this in `guildIdValidator`. You can use more than one validator in an express API route and they'll run in order of definition like so.

```js
router.put(
  "/:guildId",
  AuthController.authenticate,
  // runs guildIdValidator then updateGuildValidator
  guildIdValidator,
  updateGuildValidator,
  async (request, response) => {
  // ...
});
```